index.md

```diff
@@ -1,7 +1,18 @@
 # Mbed TLS documentation hub
 
-This is the unified documentation for the Mbed TLS project.
-This documentation is currently incomplete and will be expanded over time.
+Mbed TLS provides an open-source implementation of cryptographic primitives,
+X.509 certificate handling and the SSL/TLS and DTLS protocols. It provides a
+reference implementation of the [PSA Cryptography API](https://arm-software.github.io/psa-api/crypto/).
+The project also supports the PSA Cryptoprocessor Driver Interface which enables
+support for cryptoprocessor drivers. The small code footprint makes the project
+suitable for embedded systems. It has many users, including
+[TF-A](https://www.trustedfirmware.org/projects/tf-a/),
+[TF-M](https://www.trustedfirmware.org/projects/tf-m/) and
+[OP-TEE](https://www.trustedfirmware.org/projects/op-tee/).
```

Comment:
(note, not requesting a change: once we get to line wrapping like this, my personal preference is to always start a new line when a new sentence starts. this makes later diffs much easier to read)